bookmarks/templates/bookmarks/bulk_edit/bar.html

```diff
@@ -23,6 +23,7 @@
           <option value="bulk_unshare">Unshare</option>
         {% endif %}
         <option value="bulk_refresh">Refresh from website</option>
+        <option value="bulk_snapshot">Create HTML snapshot</option>
```

Comment:
This is only supported in the plus image, so it would be better to only show this if the feature is actually active. Here is a check that implements this when scheduling a background task: https://github.com/sissbruecker/linkding/blob/887e42a9fbc35f8ea0b5f7e3fca77b729f280dca/bookmarks/services/tasks.py#L254-L255

To make this available in the template I would suggest to expose `settings.LD_ENABLE_SNAPSHOTS` in a field in `BookmarkListContext`.

Would also be good to have tests to verify that the option is only shown when the feature is enabled.